aten/src/ATen/native/Convolution.cpp

```diff
@@ -275,13 +289,15 @@ at::Tensor _convolution(
   auto input = input_r.contiguous();
   auto weight = weight_r;
   auto bias = bias_r;
+  auto k = input.ndimension();
+  size_t dim = k - 2;
```

Comment:
Just stick to `int64_t` here and in the function. If dim is `1` or `0` you're going to get some weird error messages.